### PR TITLE
New 5.0 Characters and Missing Collei Info

### LINKED ISSUE #300: Data for 5.0 characters is missing
We still haven't added data for Kachina, Kinich and Mualani to the repo. This issue is up for grabs for whoever wants to contribute

### LINKED ISSUE #310: Missing name and description for Collei's Normal Attack
https://genshin.jmp.blue/characters/collei/

40562479b16cb74eb6c2b6cdc15019b2

### PR DESCRIPTION
**Pull request by keihachimaru**
Addition of information about Kachina, Mualani, and Kinich in english. Kinich information is not yet complete, because he has not been released yet.
Edit: Issues #300 and #310.

### CODE REVIEW

**Comment by niktekusho** [comment]
Fixes #310 😉

**Review by man90es — CHANGES_REQUESTED** [review]
Please add Kinich data from this page: https://gensh.honeyhunterworld.com/kinich_101/?lang=EN

His release date is 18 September

**Comment by keihachimaru** [comment]
Ok, I've added them. Also I added Mualani's Talents upgrades info.

File: assets/data/characters/kinich/en.json [hunk 1]
```diff
@@ -0,0 +1,303 @@
+{
+    "name": "Kinich",
+    "title": "Turnfire Hunt",
+    "vision": "Dendro",
+    "weapon": "Claymore",
+    "gender": "Male",
+    "nation": "Natlan",
+    "affiliation": "Scions of the Canopy",
+    "rarity": 5,
+    "release": "",
```
Comment (man90es) [inline assets/data/characters/kinich/en.json]: 18 September 2024

File: assets/data/characters/kinich/en.json [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    "constellation": "Chimaera Alebriius",
+    "birthday": "",
```
Comment (man90es) [inline assets/data/characters/kinich/en.json]: 11 November

**Review by man90es — CHANGES_REQUESTED** [review]

**Review by M4GICB — CHANGES_REQUESTED** [review]
It would also be good to get these new characters added to the talent material lists

**Review by man90es — APPROVED** [review]

**Comment by man90es** [comment]
@M4GICB I didn't look at the latest commit, leaving it to you since you requested it. The rest looks good to me

**Review by M4GICB — APPROVED** [review]

**Pull request merged**